integrations/whatsapp/src/conversation.ts

```diff
@@ -1,5 +1,6 @@
 import { Client, RuntimeError } from '@botpress/client'
 import { CreateConversationPayload, IntegrationContext } from '@botpress/sdk/dist/integration'
+import { IntegrationLogger } from '@botpress/sdk/dist/integration/logger'
```

Comment:
Maybe we should not import types from `@botpress/sdk/dist`? I'm thinking this is a private directory and breaking changes could happen.

Thoughts @franklevasseur ?

Reply:
I'll leave that decision to @franklevasseur since it impacts the API surface.

@AbrahamLopez10 Maybe there's a way to infer the type of the logger without exposing the type publicly?

Reply:
like SP said, you can infer its type from the props

Reply:
@franklevasseur @spg Inferred types done in https://github.com/botpress/botpress/pull/12750/commits/7a7d6c32c9a8f525eefbab6c77bb822ed5ec80d3

Also inferred some other types from a previously merged PR.

Left a TODO there so we can remember to export them from the SDK instead.